calibrators/docs/tutorials/marker_radar_lidar_calibrator.md

```diff
@@ -0,0 +1,237 @@
+</p>
+
+Once the user clicks the button, it will show like the image below.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz2.jpg" alt="rviz2" width="500">
+</p>
+
+Once the background is extracted, it will show like the image below. The user can see that the `Add lidar-radar pair` button is enabled.
+
+<p align="center">
+    <img src="../images/marker_radar_lidar_calibrator/rviz3.jpg" alt="rviz3" width="500">
+</p>
+
+Also, the following text should be shown in the console.
+
+```text
+[marker_radar_lidar_calibrator]: Finished background model initialization
+```
+
+Additionally, the user can click the topics: `lidar_background_pointcloud`, `lidar_foreground_pointcloud`, and `radar_background_pointcloud` on the left bar of the `rviz` to visualize the output of background and foreground pointcloud. As described in the [documentation](../../README.md), the `radar_reflections` is the same as the `radar_foreground_pointcloud`, which is visible as default.
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_background.jpg" alt="lidar_background" width = 700px height = 300px ></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/lidar_foreground.jpg" alt="lidar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Lidar background pointcloud.</p></td>
+    <td><p style="text-align: center;">Lidar foreground pointcloud</p></td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_background.jpg" alt="radar_background" width = 700px height = 300px></td>
+    <td><img src="../images/marker_radar_lidar_calibrator/radar_foreground.jpg" alt="radar_foreground" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Radar background pointcloud.</p></td>
+    <td><p style="text-align: center;">Radar foreground pointcloud (shown in purple lines). <br>The pointcloud appears as lines due to the radar's lack of elevation data.</p></td>
+  </tr>
+</table>
+
+### Add lidar-radar pair
+
+After the background model has been extracted, the user can carry the radar reflector (with the tripod) and place it in front of the radar sensor. Once the reflector is positioned, the user should step a few steps away from it. In the tutorial rosbag, the user will see that both the human and the radar reflector (with tripod) are identified as foreground objects in the image below.
+
+In the image, the colored points represent different lidar foreground clusters. The purple lines indicate radar foreground detections, appearing as lines due to the radar's lack of elevation data, making the z-axis unknown. The blue point is the estimated center of the radar reflector derived from the lidar pointcloud. The blue points did not appear on the human figure because the calibrator filters out clusters where the highest point in the cluster exceeds the specified threshold.
```

Comment:
`The blue points did not appear on the human` is not correct. At most there is one point per cluster no?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/6c63f903a39d642f20420cc38049e9bfad911c6a